src/docs/user/references/ScriptLanguageSupport.adoc

```diff
@@ -437,20 +437,19 @@ docker-compose --verbose up --exit-code-from helloworld
 If splitting by space is prohibitive you can specify the split regex in the <<_glossary_generic_information, generic informations>> with the
 key *docker-compose-options-split-regex*. If you supply e.g. "!SPLIT!" as value, then your *docker-compose-up-options* will need to look like this: "--option1!SPLIT!--option2".
 
-=== Docker File
+=== Dockerfile
 
-The main behavior of a Docker_File task is to first build an image and then run a container instance from it. Once the execution is done, the container is stopped and the built image is deleted.
+The default behavior of the Dockerfile task is to build an image and then start a container instance from it. Once the execution is done, the container is stopped and the built image is deleted. The build, start, stop and remove actions can be parametrized through advanced command line options which will be detailed aferward.
```

Comment:
, and remove
parametrized => tuned
aferward => afterward, but I prefer later on than afterward